plugins/vpp/ifplugin/vppcalls/vpp1901/watch_vppcalls.go

```diff
@@ -108,8 +108,7 @@ func (h *InterfaceVppHandler) WatchDHCPLeases(leasesCh chan<- *vppcalls.Lease) e
 					continue
 				}
 				lease := dhcpEvent.Lease
-				var hostMac net.HardwareAddr
-				copy(hostMac, lease.HostMac)
```

Comment:
Why remove this? Was this causing some issue for you? 

The reason why the `copy` function was used here, was because the [`net.HardwareAddr`](https://godoc.org/net#HardwareAddr) is defined as `[]byte`, slice of bytes, and slices in go are referenced by a pointer. This means that in this case if the `lease.HostMac` field in lease event was to be modified by some other goroutine it will change `hostMac` variable too. By using `copy` we copy the entire slice to a local variable, thus avoiding this situation.

Anyway.. this article probably explains it better than me: https://www.callicoder.com/golang-slices/

Reply:
I changed it. It's now OK?

Reply:
On travis you can see that it's failing with:

```     Expected
            <string>:   00
        to be equivalent to
            <string>: 10:10:20:20:30:30
```

You converted `[]byte` to string rather than using `String()` method of `net.HardwareAddr`. See the difference here: https://play.golang.org/p/Tc4aZ2UhPu3